EmpleoDotNet/Views/JobOpportunity/Detail.cshtml

```diff
@@ -46,24 +46,27 @@
                 <hr>
                 <div class="sidebar-widget" id="company">
                     <h2>Sobre la compañia</h2>
-                    <a href="@Model.CompanyUrl"><img src="@Model.CompanyLogoUrl" alt="" class="img-responsive">
+                    <a href="@Model.CompanyUrl">
+                        <img src="@Model.CompanyLogoUrl" alt="" class="img-responsive">
                     </a>
                     <h5>@Model.CompanyName</h5>
                     <p><a href="@Model.CompanyUrl" class="btn btn-primary">Mas sobre esta compañia</a></p>
+                </div>       
+                <hr>
+                <div class="sidebar-widget" id="company-jobs">
+                    <h2>Mas empleos de esta compañia</h2>
+                    <ul>
+                        @foreach (var jobOpportunity in (List<RelatedJobDto>)ViewBag.RelatedJobs)
```

Comment:
creo que se vería mejor si este casting se hiciera al principio del archivo, es decir, que en la línea 8 haya algo como `List<RelatedJobDto> relatedJobs = ViewBag.RelatedJobs`, y que aquí esta línea solo se vea `@foreach (var jobOpportunity in relatedJobs)`